<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>onlineUser</class>
 <widget class="QWidget" name="onlineUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>94</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>541</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_username">
      <property name="text">
       <string>username</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_online">
      <property name="text">
       <string>online</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_status">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_viewAllImages">
      <property name="text">
       <string>View All Images</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
